### ISSUE TITLE
[Feature Requests]: Auto-apply filter on page load / Don't download based on filters / Filter Organization & Editing

### ISSUE THREAD

**Issue by TechArtisanIO** [FreshRSS/FreshRSS#2216]
After using FreshRSS for a bit, I think it would be great if you could auto-apply (as in when you login to your FreshRSS account it defaults to a filter you specificy) a filter to a particular feed, category or even on a "app-wide" basis. Even better would better to prevent FreshRSS from downloading materials based on filters. 

Additionally, it be nice to have the capability to reorder filters on the User Queries page and be able to directly edit those filters on that page as well.

Thanks for a great app!

**Comment by Alkarex** [comment]
First implementation available in https://github.com/FreshRSS/FreshRSS/pull/2275

**Comment by TechArtisanIO** [comment]
Cool. Thanks! I'll give this a shot.

**Comment by aledeg** [comment]
@ahelton Could you split your request to be able to keep track of those? It's hard to follow through when there are more than one request in an issue. Thank you!

**Comment by aledeg** [comment]
See #2015

**Comment by math-GH** [comment]
@ahelton  How is the status? The user queries are already improved. What is left from the rest?
